### ISSUE TITLE
Wildcard support in search engine

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/FreeCENMigration#132]

**Comment by richpomfret** [comment]
Function for 1st letter of names required. For use with uncertain characters at beginning of name.

**Comment by edickens** [comment]
The problem with this is that every entry will be found if they use a Star.
But I can see why it may be needed, so perhaps we allow a question mark for one character, and then there has to be more actual characters following.

**Comment by Captainkirkdawson** [comment]
Implemented

**Comment by PatReynolds** [comment]
Is this live?  there and test3 it still says "Two letters must precede any wildcard in a surname."
test2 "You cannot use both wildcards and soundex in a search" (seached without county and place of census).

Search is for ?arshall Mary,  in Worksop, Nottinghamshire, all censuses.

**Comment by Captainkirkdawson** [comment]
@PatReynolds I closed because wildcard is there and has been for a very long time.
Wild card requires 2 leading characters and anything following is ignored. Wild card is * or ?
Wild card will never work with soundex. They are different concepts.
I note our help has NO mention on use of wildcard.
We had this discussion over on REG along time ago. 
UCF and wildcard are in conflict and cannot be merged in any sensible manner. 
That is why we introduced the UCF additional results display in REG. 
Unfortunately Ben has never completed the story and while it is 98% there it is turned off.

Hence "Wildcard support in search engine" is there.

NOW UCF is not there but do CEN transcribers use the UCF?. 

If they do then it likely should be written as a new story which would be dependant on completing the REG story first
